test/unit/warp/src/Activemask.cpp

```diff
@@ -51,7 +51,9 @@ class ActivemaskMultipleThreadWarpTestKernel
 
         auto const actual = alpaka::warp::activemask(acc);
         using Result = decltype(actual);
-        Result const allActive = (Result{1} << static_cast<Result>(warpExtent)) - 1;
+        Result const allActive = static_cast<size_t>(warpExtent) == sizeof(Result) * 8u
```

Comment:
```suggestion
        Result const allActive = static_cast<size_t>(warpExtent) == sizeof(Result) * CHAR_BIT
```
I was very confused what the `8u` meant until I noticed you wanted to convert a size in bytes to bits. With `CHAR_BIT` this is more clear and i supports architectures where 1byte != 8bit :)